### PR TITLE
added md files for new text detection models

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
@Ilya-Krylov please retarget PR to develop branch

File: demos/text_detection_demo/README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 The demo shows an example of using neural networks to detect and recognize printed text rotated at any angle in various environment. You can use the following pre-trained models with the demo:
 
-* `text-detection-0002`, which is a detection network for finding text.
+* `text-detection-0003`, which is a detection network for finding text.
+* `text-detection-0004`, which is a light-weight detection network for finding text.
```
Comment (IRDonch) [inline demos/text_detection_demo/README.md]: "lightweight" is spelled without a hyphen.

**Comment by Ilya-Krylov** [comment]
@vladimir-dudnik @IRDonch done

**Comment by kchechil** [comment]
Jenkins please retry a build

File: intel_models/index.md [hunk 2]
```diff
@@ -139,7 +139,8 @@ Deep Learning models for text detection in various applications
 
 | Model Name                                                                                                                                     | Complexity (GFLOPs)  | Size (Mp)  |
 |------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
-| [text-detection-0002](./text-detection-0002/description/text-detection-0002.md)                                                                | 51.256               | 6.747      |
+| [text-detection-0003](./text-detection-0003/description/text-detection-0003.md)                                                                | 51.256               | 6.747      |
+| [text-detection-0004](./text-detection-0003/description/text-detection-0004.md)                                                                | TBD                  | TBD        |
```
Comment (kchechil) [inline intel_models/index.md]: @Ilya-Krylov there is a bug in the path, CI job failed due to that
19:20:26  FILE: index.md
19:20:27  [✖] ./text-detection-0003/description/text-detection-0004.md
19:20:27  
19:20:27  ERROR: dead links found!
19:20:27  [Pipeline] echo
19:20:27  [Error] Found broken links in index.md

Comment (zmaslova) [inline intel_models/index.md]: `./text-detection-0003/` -> `./text-detection-0004/`

**Comment by Ilya-Krylov** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Comment by kchechil** [comment]
Jenkins please retry a build

**Pull request merged**